doc/source/whatsnew/v1.0.0.rst

```diff
@@ -214,7 +214,7 @@ Other
 - Trying to set the ``display.precision``, ``display.max_rows`` or ``display.max_columns`` using :meth:`set_option` to anything but a ``None`` or a positive int will raise a ``ValueError`` (:issue:`23348`)
 - Using :meth:`DataFrame.replace` with overlapping keys in a nested dictionary will no longer raise, now matching the behavior of a flat dictionary (:issue:`27660`)
 - :meth:`DataFrame.to_csv` and :meth:`Series.to_csv` now support dicts as ``compression`` argument with key ``'method'`` being the compression method and others as additional compression options when the compression method is ``'zip'``. (:issue:`26023`)
-
+- Reindexing a `PeriodIndex` with an index of dtype `Object` no longer uses the `PeriodIndex`'s `_int64index`
```

Comment:
can you use ``:meth:`PeriodIndex` `` here, double back-ticks around ``object`` (and make it lowercase); also can you make this a bit more user friendly, no need to mention _int64index; these are user facing notes.

also add the issues that are closed.

move to the Indexing section.